startup_configs/leaf1.conf

```diff
@@ -20,7 +20,7 @@
     subinterface 0 {
         ipv4 {
             admin-state enable
-            address 10.0.0.1/32 !!! Pattern 10.0.0.<leaf ID>
+            address 10.0.0.1/32 !!! Pattern 10.0.0.[leaf ID]
```

Comment:
I got the following error:
```
02:17:03 ERRO failed to run postdeploy task for node leaf1: command execution error:At line 23: Parsing error: While parsing '>': Missing value for 'file'
```

Replaces `<>` with `[ ]`. Is there a better way?